### PR TITLE
add new relic browser monitoring via ember-new-relic

### PR DESCRIPTION
**Pull request by hakubo**
# PLEASE don't merge unless you are LCF

## Description

New Relic browser recently added support for SPA applications

With this addon we can:
- monitor performance of our app
- log all JS errors (even the once coming from Ember and not hitting window.onerror)
## Reviewers

@Wikia/x-wing @ljagiello

### CODE REVIEW

File: front/main/package.json [hunk 1]
```diff
@@ -53,6 +53,7 @@
     "ember-getowner-polyfill": "1.0.0",
     "ember-in-viewport": "pauloslund/ember-in-viewport.git#347251dc727956c95d1f9f6b8c7c64c7da11cd34",
     "ember-load-initializers": "0.5.1",
+    "ember-new-relic": "0.2.0",
```
Comment (Grunny) [inline front/main/package.json]: Should be added to shrinkwrap in this PR as well?

Comment (hakubo) [inline front/main/package.json]: done

**Comment by Grunny** [comment]
LGTM.

**Comment by rogatty** [comment]
LGTM

**Pull request merged**